apps/cyberstorm-storybook/constants.ts

```diff
@@ -1,2 +1,5 @@
 export const API_DOMAIN =
   process.env.NEXT_PUBLIC_API_DOMAIN || "https://thunderstore.io";
+
+export const ROOT_DOMAIN =
```

Comment:
If this is needed in Storybook, should the storybook app's LinkLibrary be uodated too?

Reply:
Yes but I'm deprioritizing storybook for now. I'm barely using it and as it is, it really does not give any value to the dev pipeline right now IMHO. In addition the following PRs in this stack will majorly break storybook, since we'll be moving the layouts to be under NextJS.

I'll have to revisit storybook as a separate task.

tbh, not sure why I even tried to keep storybook working in this PR.

Reply:
@Oksamies Well you shouldn't be breaking things just because you don't use them. Other people exist too, you know?

And if you think Storybook is not needed, the way forward is to discuss it with the team and drop it altogether, rather than keep it in the repo in broken state. The current approach comes across sloppy and unprofessional.